applicationinsights/channel/SenderBase.py

```diff
@@ -115,7 +115,7 @@ def send(self, data_to_send):
 
         request = HTTPClient.Request(self._service_endpoint_uri, bytearray(request_payload, 'utf-8'), { 'Accept': 'application/json', 'Content-Type' : 'application/json; charset=utf-8' })
         try:
-            response = HTTPClient.urlopen(request)
+            response = HTTPClient.urlopen(request, timeout=10)
```

Comment:
There doesn't seem to be one for this SDK (I don't regularly use this SDK and hadn't realized this!). However, it seems that if you make a class method on SenderBase to change the timeout value, it would be accessible from tc.channel.sender